packages/styles/radio-card.css

```diff
@@ -43,7 +43,7 @@
   min-height: 44px;
 }
 .RadioCardGroup__Icon.Icon svg {
-  color: var(--accent-success-dark);
+  color: #4f990f;
```

Comment:
A comment here for posterity would be helpful to explain why this one-off exists.